modules/cli/pages/cbstats/cbstats-collections-details.adoc

```diff
@@ -22,89 +22,94 @@ The optional `vbucket-reference` parameter is an integer, in the range of `0` to
+Each `scopeID` or `collectionID` is a hexadecimal number, prefixed by `0x`.
+Each `datumLabel` is a string.
+Each data key is concluded with a colon, and the associated value is presented to the right of the colon.
 
-Subsequent information in the output relates to each vBucket corresponding to `testBucket`.
+The initial two lines provide information that identifies existing collections.
+Two are listed, which are the `_default` collection, and `MyCollection`.
+Each is identified with hexadecimal numbers: the `_default` collection is identified as `0x0` (residing within scope `0x0`); and `MyCollection` as `0x9` (residing within scope `0x8`).
+
+(Note that the `_default` collection is indeed provided by default, within the `_default` scope, for every Couchbase or Ephemeral bucket.
+The collection `MyCollection` is one that has been custom-created by the administrator, and does not appear in the sample bucket `travel-sample` by default.)
+
+The `manifest_uid` value (here, `4`) is associated with the current state of collections on the node; and will be incremented whenever a collections-related change is made.
+
+Subsequent information in the output relates to each vBucket corresponding to `travel-sample`, on the current node.
 The vBuckets are numbered, from `0` to `1023`.
 For each vBucket, information is provided on sequence numbers and total items.
 
 The following command specifies a `vbucket-reference`, thereby returning information on the specified vBucket only:
 
 ----
-/opt/couchbase/bin/cbstats -u Administrator -p password \
--b testBucket localhost:11210 collections-details 8
+/opt/couchbase/bin/cbstats localhost:11210 \
+-u Administrator -p password \
+-b travel-sample collections-details 8
 ----
 
 Details for vBucket `8` are duly returned:
 
 ----
-vb_8:collection:0x0:entry:high_seqno:           0
-vb_8:collection:0x0:entry:items:                0
-vb_8:collection:0x0:entry:persisted_high_seqno: 0
-vb_8:collection:0x0:entry:scope:                0
-vb_8:collection:0x0:entry:start_seqno:          0
-vb_8:collection:0x8:entry:high_seqno:           2
-vb_8:collection:0x8:entry:items:                0
-vb_8:collection:0x8:entry:persisted_high_seqno: 2
-vb_8:collection:0x8:entry:scope:                8
-vb_8:collection:0x8:entry:start_seqno:          2
-vb_8:collection:0x9:entry:high_seqno:           3
-vb_8:collection:0x9:entry:items:                0
-vb_8:collection:0x9:entry:persisted_high_seqno: 3
-vb_8:collection:0x9:entry:scope:                8
-vb_8:collection:0x9:entry:start_seqno:          3
-vb_8:collection:0xa:entry:high_seqno:           4
-vb_8:collection:0xa:entry:items:                0
-vb_8:collection:0xa:entry:persisted_high_seqno: 4
-vb_8:collection:0xa:entry:scope:                0
-vb_8:collection:0xa:entry:start_seqno:          4
-vb_8:manifest:default_exists:                   true
-vb_8:manifest:entries:                          4
+vb_8:0x0:high_seqno:           26
+vb_8:0x0:items:                26
```

Comment:
I think with the description you have put earlier in this section, a reader should know this 0x0 is the collection ID, but should we look to improve the code here and return something more self describing - "cid_0x0"?

Reply:
"vb_8:cid_0x0:label    value"

Reply:
Yes, I think it will help to be specific.